src/modules/universal-discovery/components/finder/finder.component.js

```diff
@@ -104,18 +208,52 @@ export default class FinderComponent extends Component {
                 const location = activeLocations[key];
 
                 if (location.parent === parentLocationId) {
-                    const results = [...location.data.View.Result.searchHits.searchHit, ...data.View.Result.searchHits.searchHit];
+                    const results = [...location.data, ...data.View.Result.searchHits.searchHit];
 
                     location.offset = offset;
-                    location.data.View.Result.searchHits.searchHit = results;
-                    locationsMap[parentLocationId] = location.data;
+                    location.data = results;
+                    locationsMap[parentLocationId] = location;
                 }
             });
 
             return Object.assign({}, state, { activeLocations, locationsMap });
         });
     }
 
+    /**
+     * Loads branch children (sub-items)
+     *
+     * @method loadBranchChildren
+     * @param {String} parent the parent id
+     * @memberof FinderComponent
+     */
+    loadBranchChildren(parent) {
+        const promise = new Promise(resolve => this.props.findLocationsByParentLocationId(
+            Object.assign({}, this.props.restInfo, {parentLocationId: parent}), resolve));
```

Comment:
Strange line break.

Reply:
I follow the pattern in this file.

Reply:
Almost good, but the closing parenthesis should be in a new line. I think it's a good start to include Prettier in our development toolset. It will help us avoiding discussions about CS ;)